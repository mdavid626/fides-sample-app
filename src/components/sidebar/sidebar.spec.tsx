import React from 'react';
import { render } from '@testing-library/react';
import Sidebar from './sidebar';
import { BrowserRouter, createBrowserRouter } from 'react-router-dom';

const renderWithRouter = (component: JSX.Element) =>
  render(component, {
    wrapper: ({ children }) => <BrowserRouter>{children}</BrowserRouter>,
  });

describe('sidebar', () => {
  it('should render', () => {
    const { asFragment } = renderWithRouter(<Sidebar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
