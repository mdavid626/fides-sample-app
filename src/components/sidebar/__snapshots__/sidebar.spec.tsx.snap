// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sidebar should render 1`] = `
<DocumentFragment>
  <div>
    <a
      href="/"
    >
      Movies
    </a>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</DocumentFragment>
`;
